egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -160,9 +169,9 @@ def AddOutputLayer(config_lines, input, label_delay=None, suffix=None):
         name = '{0}-{1}'.format(name, suffix)
 
     if label_delay is None:
-        component_nodes.append('output-node name={0} input={1}'.format(name, input['descriptor']))
+        component_nodes.append('output-node name={0} input={1} objective={2}'.format(name, input['descriptor'], objective_type))
     else:
-        component_nodes.append('output-node name={0} input=Offset({1},{2})'.format(name, input['descriptor'], label_delay))
+        component_nodes.append('output-node name={0} input=Offset({1},{2}) objective={3}'.format(name, input['descriptor'], label_delay, objective_type))
```

Comment:
If you have the time yes.

On Wed, Mar 2, 2016 at 2:38 AM, Vimal Manohar <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/nnet3/components.py
> <https://github.com/kaldi-asr/kaldi/pull/539#discussion_r54688606>:
>
> >      else:
> > -        component_nodes.append('output-node name={0} input=Offset({1},{2})'.format(name, input['descriptor'], label_delay))
> > +        component_nodes.append('output-node name={0} input=Offset({1},{2}) objective={3}'.format(name, input['descriptor'], label_delay, objective_type))
>
> The rest of components.py does not have named arguments. Should I modify
> all of it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/539/files#r54688606>.
>